README.md

```diff
@@ -42,9 +42,10 @@ Navigate to your Jekyll project directory and create a Guardfile using:
 Or if you already have a Guardfile, add a Jekyll guard.
 
 ```ruby
-guard "jekyll_plus" do
+ignore /^_site/ # NOTE: this can interfere with Guard::LiveReload
```

Comment:
This could be a feature request in `Listen` (guard-clause-local ignores) - though it's probably not trivial to implement.